deployments/container/Makefile

```diff
@@ -80,7 +80,11 @@ endif
 endif
 
 build-%: DIST = $(*)
+ifneq ($(SAMPLE),nvbandwidth)
 build-%: DOCKERFILE = $(CURDIR)/deployments/container/Dockerfile.$(DOCKERFILE_SUFFIX)
+else
+build-%: DOCKERFILE = $(CURDIR)/deployments/container/$(SAMPLE)/Dockerfile.$(DOCKERFILE_SUFFIX)
+endif
```

Comment:
Could we modify the IMAGE_TAG here, I don't think `nvbandwidth-8169f9fa-ubuntu22.04` is a good tag for the nvbandwidth image, maybe we want `nvbandwidth-8169f9fa`

Reply:
we need the nvbandwidth and cuda_version tag actually. these images are version sensitive.

Reply:
Yeah I discussed this with @klueska

Reply:
We can update the tags to be whatever we want them to be. Please remember that:

* The `VERSION` for the images released from this repo is `cuda12.6.2` for example.
* The tag should be different for each build (e.g. SHA) so that we can test early access bits.
* The image will be released when tagging the (internal) repo. Currently we tag with `cuda<VERSION>` since the base images are the main driver for updates.